model/multimap.go

```diff
@@ -0,0 +1,69 @@
+package model
```

Comment:
I believe this would be more suited for the `store` package, as part of the `map.go` file...

Reply:
I think it should stay in `model` because it is a helper datastructure. It is not related to `store` or `mip` but lives next to them. In that regard it is similar to `Domain` which also has no connection to any paradigm.